spec/models/plant_accession_spec.rb

```diff
@@ -47,6 +47,25 @@
     end
   end
 
+  context 'unique attributes' do
+    it 'disallows duplicate plant_accession and originating_organisation' do
```

Comment:
It is easier to use [shoulda-matchers](https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/active_record/validate_uniqueness_of_matcher.rb) gem to test validation (it is already added to the project).